docker build errors in CI

`docker-container` buildx driver in CI jobs fails when exporting to cache.

https://github.com/startersclan/hlstatsx-community-edition/actions/runs/6730862427/job/18294381391#step:7:1163

```sh
#46 [daemon] exporting to docker image format
#46 sending tarball 19.6s done
could not lock /tmp/.buildx-cache-web/index.json.lock
#46 CANCELED

#45 [awards] exporting to docker image format
#45 sending tarball 20.0s done
#45 CANCELED
Error: Process completed with exit code 17.
```

--- a/docker-compose.build.yml
+++ b/docker-compose.build.yml
@@ -4,26 +4,26 @@ services:
     build:
       cache_from:
         - type=local,src=/tmp/.buildx-cache-daemon
-      cache_to:
-        - type=local,dest=/tmp/.buildx-cache-daemon,mode=max
+      # cache_to:
+      #   - type=local,dest=/tmp/.buildx-cache-daemon,mode=max
 
   awards:
     build:
       cache_from:
         - type=local,src=/tmp/.buildx-cache-daemon
-      cache_to:
-        - type=local,dest=/tmp/.buildx-cache-daemon,mode=max
+      # cache_to:
+      #   - type=local,dest=/tmp/.buildx-cache-daemon,mode=max
 
   web:
     build:
       cache_from:
         - type=local,src=/tmp/.buildx-cache-web
-      cache_to:
-        - type=local,dest=/tmp/.buildx-cache-web,mode=max
+      # cache_to:
+      #   - type=local,dest=/tmp/.buildx-cache-web,mode=max
 
   heatmaps:
     build:
       cache_from:
         - type=local,src=/tmp/.buildx-cache-web
-      cache_to:
-        - type=local,dest=/tmp/.buildx-cache-web,mode=max
+      # cache_to:
+      #   - type=local,dest=/tmp/.buildx-cache-web,mode=max

--- a/docker-compose.example.build.yml
+++ b/docker-compose.example.build.yml
@@ -7,8 +7,8 @@ services:
       target: prod
       cache_from:
         - type=local,src=/tmp/.buildx-cache-daemon
-      cache_to:
-        - type=local,dest=/tmp/.buildx-cache-daemon,mode=max
+      # cache_to:
+      #   - type=local,dest=/tmp/.buildx-cache-daemon,mode=max
 
   awards:
     build:
@@ -17,8 +17,8 @@ services:
       target: prod
       cache_from:
         - type=local,src=/tmp/.buildx-cache-daemon
-      cache_to:
-        - type=local,dest=/tmp/.buildx-cache-daemon,mode=max
+      # cache_to:
+      #   - type=local,dest=/tmp/.buildx-cache-daemon,mode=max
 
   web:
     build:
@@ -27,8 +27,8 @@ services:
       target: prod
       cache_from:
         - type=local,src=/tmp/.buildx-cache-web
-      cache_to:
-        - type=local,dest=/tmp/.buildx-cache-web,mode=max
+      # cache_to:
+      #   - type=local,dest=/tmp/.buildx-cache-web,mode=max
 
   heatmaps:
     build:
@@ -37,5 +37,5 @@ services:
       target: prod
       cache_from:
         - type=local,src=/tmp/.buildx-cache-web
-      cache_to:
-        - type=local,dest=/tmp/.buildx-cache-web,mode=max
+      # cache_to:
+      #   - type=local,dest=/tmp/.buildx-cache-web,mode=max

